src/server/services/wms/qgswmsparameters.h

```diff
@@ -201,6 +201,14 @@ namespace QgsWms
        */
       bool isValid() const override;
 
+      /**
+       * Converts the parameter into a list of strings and keeps empty parts
+       * Default style value is an empty string
+       * \param delimiter The character used for delimiting
+       * \returns A list of strings
+       */
```

Comment:
\since

Reply:
\since version 3.8 or 3.4.7 ?

If i use backport-bot the brench 3.4 will have information about 3.8, isn't it ?

Reply:
Done